lms/djangoapps/instructor/features/data_download.py

```diff
@@ -18,7 +18,7 @@ def find_student_profile_table(step):  # pylint: disable=unused-argument
     world.wait_for_visible('#data-student-profiles-table')
 
     # Wait for the data table to be populated
-    world.wait_for(lambda _: world.css_text('#data-student-profiles-table') not in [u'', u'Loading...'])
+    world.wait_for(lambda _: world.css_text('#data-student-profiles-table') not in [u'', u'Loading...', u'Loading'])
```

Comment:
We've generally been moving away from putting "..." on UI strings. So it probably was Loading... at one point.

Can we just not change it to the actual string that is displaying?